jmix-flowui/flowui/src/main/java/io/jmix/flowui/sys/UiAccessChecker.java

```diff
@@ -78,6 +78,19 @@ public boolean isViewPermitted(Class<?> target) {
         return hasAccess;
     }
 
+    /**
+     * Checks for access to the view.
+     * Throws exception if no access.
+     *
+     * @param target class to check
+     * @throws AccessDeniedException if the user does not have access to the view
+     */
+    public void checkViewPermitted(Class<?> target) {
```

Comment:
Why did you introduce a new method here? Why not to use an existing `isViewPermitted()`? As I see this method is used only once inside the `AbstractWindowBuilderProcessor`. Why not to write these lines there?

```java
        if (!accessChecker.isViewPermitted(target)) {
            throw new AccessDeniedException("view", target.getName());
        }
```

Reply:
A handler is created for `AccessDeniedException`, I find it convenient to have one method that can potentially be used in other parts of the application.

Reply:
Now it is only used in one place, but maybe it can be used somewhere else in future. If Gleb is fine with the new method in "UiAccessChecker", then let it be.